src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -203,9 +205,10 @@
 	// Unset ca and cert if not required to avoid storing in config
 	if ($method == "pre_shared_key" || $method == "xauth_psk_server") {
 		unset($pconfig['certref']);
+		unset($pconfig['pkcs11certref']);
 	}
 
-	if ($method != "cert" && $method != "xauth_cert_server" && $method != "eap-tls") {
+	if ($method != "cert" && $method != "xauth_rsa_server" && $method != "eap-tls" && $method != "pkcs11") {
```

Comment:
This should be `xauth_cert_server` but got reverted to `_rsa_`

Also would probably be easier to read as an `in_array()` as was done with the ipsec.inc line ~2426-2427 change

Reply:
done